cmd/process-agent/collector.go

```diff
@@ -268,6 +279,11 @@ func (l *Collector) messagesToResults(start time.Time, name string, messages []m
 			extraHeaders.Set(headers.EVPOriginVersionHeader, version.AgentVersion)
 		}
 
+		switch m.(type) {
```

Comment:
I guess we don't plan to activate the Manifest collection without the orchestration collection, right? If yes, let's move that to above under the orchestration check